app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -282,6 +302,14 @@ public String getDescription() {
         return description;
     }
 
+    public String getCaption() {
```

Comment:
Javadoc for all of these methods too. You can copy/paste from the properties.
Source code is written by 1 person, read by 1000 persons. So better document well, even if you think that the reader could find the info elsewhere :-)

Reply:
Okay I'll add the doc

Reply:
Added the documentation here and also in other relevant places related to captions